README.md

```diff
@@ -43,6 +43,8 @@ Then run:
 npm install
 npm run compile
 npm run migrate -- --network rinkeby
+# if you want to deploy the markets with another account (recommended), you will need to add those accounts as admins
+npm run migrate -- --network rinkeby --admins=address1,...,address2 (comma separated)
```

Comment:
I wonder if we should throw the `--reset` flag in there as well: I think I had to do so when deploying to Rinkeby.

Also, this means that GnosisDB should be set up before setting up these contracts in Apollo, right?

Reply:
Yeah, the first step should be create the contracts, then set up GnosisDB